prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -448,7 +457,15 @@ workflow_has_parameter AOD && add_W o2-aod-producer-workflow "--info-sources $AO
 
 # ---------------------------------------------------------------------------------------------------------------------
 # Quality Control
-workflow_has_parameter QC && { source $MYDIR/qc-workflow.sh; [[ $? != 0 ]] && exit 1; }
+if [[ ! -z $QC_JSON_FROM_OUTSIDE ]]; then
+    if [[ -z $QC_CONFIG_PARAM ]]; then
+        QC_CONFIG_PARAM="--local --host $QC_HOST"
+    fi
+    echo "qc json from outside $QC_JSON_FROM_OUTSIDE"
+    workflow_has_parameter QC && add_W o2-qc "--config json://$QC_JSON_FROM_OUTSIDE $QC_CONFIG_PARAM" "" 0
+else
+    workflow_has_parameter QC && { source $O2DATAPROCESSING_ROOT/production/qc-workflow.sh; [[ $? != 0 ]] && exit 1; }
+fi
```

Comment:
There should be no echo, or it should go to stderr.
If you want to provide multiple parameters, why don't you just provide a full string providing multiple parameters to `o2-qc`? That would be simpler and single-line.

Reply:
Thinking a bit about it, I think it makes actually sense to have multiple variables. But then I would align it between the JSON_FROM_OUTSIDE case here, and what is done in qc-workflow.sh.
How about we use
```
add_W o2-qc "--config json://$QC_JSON_FROM_OUTSIDE ${QC_CONFIG_PARAM:---local --host ${QC_HOST:-localhost}}" "" 0
```
here (also a bit shorter :). Sorry, but I try to keep the script short), and then
```
"--config json://$MERGED_JSON_FILENAME ${QC_CONFIG_PARAM:---local --host ${QC_HOST:-localhost}}"
```
in O2DataProcessing/production/qc-workflow.sh

Reply:
Actually, I just added the suggested simplification and also the QC_JSON_FROM_OUTSIDE handling to qc-workflow.sh:
AliceO2Group/O2DataProcessing#125. If that makes sense for you, I'd merge it today and bump it in alidist, so it will go in tomorrow's tag.

Reply:
OK, I did some more fixes in the O2DataProcessing part, so I just went ahead and merged the PR. QC_JSON_FROM_OUTSIDE is not supported there in any case.

Reply:
Thanks David, perfect! I merged also your PR to my branch here, so now it should be fine, I guess.